client/src/features/bulletins/Bulletins/Filters/DoubleSlider/DoubleSlider.tsx

```diff
@@ -0,0 +1,85 @@
+import React, { useRef, useEffect } from 'react';
+import './DoubleSlider.scss';
+import { CustomInput } from 'reactstrap';
+
+type Props = {
+  setMinPrice: (minPrice: number) => void;
+  setMaxPrice: (maxPrice: number) => void;
+};
+
+export const DoubleSlider = React.memo(({ setMinPrice, setMaxPrice }: Props) => {
+  const minSlider = useRef<HTMLInputElement>(null);
+  const minBubble = useRef<HTMLInputElement>(null);
+
+  const maxSlider = useRef<HTMLInputElement>(null);
+  const maxBubble = useRef<HTMLInputElement>(null);
+
+  useEffect(() => {
+    updateBubbles(minSlider.current!, minBubble.current!);
+    updateBubbles(maxSlider.current!, maxBubble.current!);
+  }, []);
+
+  const updateBubbles = (cursorRef: HTMLInputElement, bubbleRef: HTMLInputElement) => {
```

Comment:
In questo componente hai deciso di praticamente abbandonare React, se lo scrivevi in puro JS facevi prima ed era più pulito. Chiaramente non hai capito come si ragiona in React, consiglio di leggerti la doc ufficiale su [Thinking in React](https://reactjs.org/docs/thinking-in-react.html).